e2e/wdio/test/helpers/edit-configs.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+
+class EditConfigs {
+  editWebConfig(filepath, serverSideRenderingEndpointUrl) {
```

Comment:
Could this be more generic (as the name implies - e.g. passing the attribute name and value)? Seems this is currently only applicable to the `serverSideRenderingEngineEndpointUrl` attribute.